### DISCUSSION TITLE
Can't call New-PnSSite when Microsoft.Graph.Authentication is loaded

### DISCUSSION THREAD

**Discussion by Ambergahill** [pnp/powershell#2906] [General]
### Reporting an Issue or Missing Feature
When calling New-PnPSite and the Microsoft.Graph.Authentication module is loaded, you get an error saying "Could not load file or assembly System.IdentityModel.Tokens.Jwt Version 6.12.2.0

### Expected behavior 
The new SPO site should be created without error. Code is working fine when the Microsoft.Graph.Authentication module is not loaded

### Actual behavior 
See screenshot
![error](https://user-images.githubusercontent.com/92441493/225630873-802e67fa-ec26-4468-8072-e597cc28e5af.png)

### Steps to reproduce behavior
1. Load Microsoft.Graph.Authentication module. Currently testing with 1.23.0 but I have tested other versions with the same error.
2. Load PnP.PowerShell module. Currently testing with 2.0.41-nightly, but I have tries other versions as well.
3. Connect to Microsoft Graph using Connect-MgGraph (not sure this step is actually necessary, but in Azure functions, it doesn't look like it loads the module until it sees you are using at least one command from that module)
4. Connect to SPO using Connect-PnPOnline
5. Try to create a site with New-PnPSite

### What is the version of the Cmdlet module you are running?
See above 

### Which operating system/environment are you running PnP PowerShell on?
- [ ] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [X] Azure Functions
- [ ] Other : please specify

**Comment by gautamdsheth** [comment]
Moving it to discussion forum.

This issue is something related to loading external libraries in the same session as PnP PowerShell. This not something we maybe able to fix.

**Comment by outorted** [comment]
Hi @gautamdsheth I think this might be the same issue as reported in https://github.com/pnp/powershell/issues/3637. It seems that MgGraph and PnP are not compatible with each other in terms of authentication. Is there anything that can be done from your end? Microsoft is moving everything towards MgGraph (AzureAD is now obsolete) and this conflict has broken many of my scripts.
